src/wp-includes/script-loader.php

```diff
@@ -1009,7 +1009,7 @@ function wp_default_scripts( $scripts ) {
 		/* translators: %s: File name. */
 		'error_uploading'           => __( '&#8220;%s&#8221; has failed to upload.' ),
 		'unsupported_image'         => __( 'This image cannot be displayed in a web browser. For best results convert it to JPEG before uploading.' ),
-		'noneditable_image'         => __( 'This image cannot be processed by the web server. Convert it to JPEG or PNG before uploading.' ),
+		'noneditable_image'         => __( 'This image cannot be processed by the web server to generate responsive image sizes. Convert it to JPEG or PNG before uploading.' ),
```

Comment:
I think this is easier to read, and more concise as well:
```
The web server cannot generate responsive image sizes for this image. Convert it to JPEG or PNG before uploading.
```

Reply:
Applied!